server/db/models/User.js

```diff
@@ -10,65 +10,91 @@ const User = db.define('user', {
   username: {
     type: Sequelize.STRING,
     unique: true,
-    allowNull: false
+    allowNull: false,
+    validate: {
+      notEmpty: true,
+      isEmail: true,
+    },
   },
   password: {
     type: Sequelize.STRING,
-  }
-})
+  },
+  // name: {
```

Comment:
Make sure to get rid of commented code in your main branch. Anything that makes it to your main branch should generally be free of commented code and console.logs that were used for debugging. The more stuff hanging around that doesn't serve a purpose anymore, the harder it becomes to maintain your repository long term